src/urllib3/util/timeout.py

```diff
@@ -19,20 +19,26 @@
 class Timeout(object):
     """Timeout configuration.
 
-    Timeouts can be defined as a default for a pool::
+    Timeouts can be defined as a default for a pool:
```

Comment:
Those code blocks don't show up in the doc: https://urllib3--1966.org.readthedocs.build/en/1966/reference/urllib3.util.html?highlight=timeout#urllib3.util.Timeout

Reply:
Ah, I think I'm missing a `:` after `code-block:`